modules/examples/src/main/java/com/ibm/cloud/platform_services/iam_identity/v1/IamIdentityExamples.java

```diff
@@ -408,6 +441,342 @@ public static void main(String[] args) throws Exception {
                     e.getStatusCode(), e.getMessage(), e.getDebuggingInfo()), e);
         }
 
+        try {
+            System.out.println("createProfile() result:");
+
+            // begin-create_profile
+
+            CreateProfileOptions createProfileOptions = new CreateProfileOptions.Builder()
+                    .name(profileName)
+                    .description("Example Profile")
+                    .accountId(accountId)
+                    .build();
+
+            Response<TrustedProfile> response = service.createProfile(createProfileOptions).execute();
+            TrustedProfile profile = response.getResult();
+            profileId = profile.getId();
+            profileIamId = profile.getIamId();
+
+            System.out.println(profile);
+
+            // end-create_profile
+
+        } catch (ServiceResponseException e) {
+            logger.error(String.format("Service returned status code %s: %s\nError details: %s",
+                    e.getStatusCode(), e.getMessage(), e.getDebuggingInfo()), e);
+        }
+
+        try {
+            System.out.println("getProfile() result:");
+
+            // begin-get_profile
+
+            GetProfileOptions getProfileOptions = new GetProfileOptions.Builder()
+                    .profileId(profileId)
+                    .build();
+
+            Response<TrustedProfile> response = service.getProfile(getProfileOptions).execute();
+            TrustedProfile profile = response.getResult();
+            profileEtag = response.getHeaders().values("Etag").get(0);
+
+            System.out.println(profile);
+
+            // end-get_profile
+
+        } catch (ServiceResponseException e) {
+            logger.error(String.format("Service returned status code %s: %s\nError details: %s",
+                    e.getStatusCode(), e.getMessage(), e.getDebuggingInfo()), e);
+        }
+
+        try {
+            System.out.println("getListProfile() result:");
```

Comment:
```suggestion
            System.out.println("listProfile() result:");
```

Reply:
This way the consistency across SDKs is ensured. Possibly this file was generated before the method name changed to `listProfile`.